src/Swift.Bindings/tests/IntegrationTests/Swift.Bindings.Integration.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net9.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>disable</Nullable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+    <PreTestCommand>cd $(OutputPath)$(TargetFramework) &amp;&amp; cmake $(MSBuildThisFileDirectory) &amp;&amp; make</PreTestCommand>
+  </PropertyGroup>
+
+  <Target Name="CompileNativeLibs" BeforeTargets="PrepareForBuild" Condition="'$(OS)' != 'Windows_NT'">
+    <!-- Generate Swift libraries and projection tooling bindings -->
+    <Exec Command="$(PreTestCommand)" />
+
+    <!-- Compile generated bindings with tests -->
+    <ItemGroup>
+      <Compile Remove="$(OutputPath)/**/*.cs" />
+      <Compile Include="$(OutputPath)/**/*.cs" />
```

Comment:
What's the purpose of removing all files, then just re-adding them back?

Reply:
Correct, not needed.